src/main/java/view/planning/EventPlanningWindow.java

```diff
@@ -112,12 +112,19 @@ public String getEventName() {
 
     @Override
     public LocalDateTime getEventDate() {
-        return LocalDateTime.now(); //TODO replace with datepicker
+        Date tempDate = dateTimePicker.getDate();
+        LocalDateTime eventDateTime = LocalDateTime.ofInstant(tempDate.toInstant(), ZoneId.systemDefault());
+        return eventDateTime;
     }
 
     @Override
     public List<UserView> getUsers() {
-        return List.of(); //TODO return userviews with things
+        List<UserView> listOfUsers = new ArrayList<>();
```

Comment:
`userViews` is okay. You don't need to place `list` word anywhere in variable's name because it is contained in the type

Reply:
Ok, understand it. Changed name to usersView